setup.py

```diff
@@ -61,7 +61,6 @@
     'numpy>=1.9.2',
     'pandas>=0.16.1',
     'scipy>=0.15.1',
-    'bottleneck>=1.0.0'
```

Comment:
I think bottleneck needs to be required at the moment. We [discovered](https://github.com/quantopian/empyrical/pull/45#issuecomment-299338946) that it has diverged slightly from numpy, and we rely on the bottleneck behavior.